LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-invalid.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Level 3: parsing align-tracks with invalid values</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-3/#tracks-alignment">
+<meta name="assert" content="align-tracks supports only the grammar 'normal | <baseline-position> | <content-distribution> | <overflow-position>? <content-position>'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_invalid_value("align-tracks", "auto");
+test_invalid_value("align-tracks", "baseline last");
+test_invalid_value("align-tracks", "center baseline");
+test_invalid_value("align-tracks", "first");
+test_invalid_value("align-tracks", "flex-start flex-end");
+test_invalid_value("align-tracks", "last");
+test_invalid_value("align-tracks", "left");
+test_invalid_value("align-tracks", "legacy center");
+test_invalid_value("align-tracks", "legacy left");
+test_invalid_value("align-tracks", "legacy");
+test_invalid_value("align-tracks", "normal baseline");
+test_invalid_value("align-tracks", "right legacy");
+test_invalid_value("align-tracks", "safe self-end");
+test_invalid_value("align-tracks", "safe");
+test_invalid_value("align-tracks", "self-end unsafe");
+test_invalid_value("align-tracks", "self-start");
+test_invalid_value("align-tracks", "start safe");
+test_invalid_value("align-tracks", "unsafe right");
+test_invalid_value("align-tracks", "unsafe");
+
+test_invalid_value("align-tracks", "auto, left");
+test_invalid_value("align-tracks", "foo, auto");
```

Comment:
I think ones like `start, left, end`/`left, start, end`/`start, end, left` would be worth adding too